.github/CODEOWNERS

```diff
@@ -106,17 +112,22 @@
 # PRLabel: %Communication - Common
 /sdk/communication/**/_shared/                                       @Azure/acs-identity-sdk @AikoBB @maximrytych-ms @mjafferi-msft
 
+# ServiceOwners: @lynshi
+# ServiceLabel: %Confidential Ledger
 # PRLabel: %Confidential Ledger
-/sdk/confidentialledger/azure-confidentialledger/                    @PallabPaul
+/sdk/confidentialledger/azure-confidentialledger/                    @PallabPaul @lynshi
 
+# ServiceOwners: @mengaims
+# ServiceLabel: %Content Safety
 # PRLabel: %Content Safety
 /sdk/contentsafety/                                                  @mengaims @JieZhou000
 
 # AzureSDKOwners: @mccoyp
```

Comment:
```suggestion
# AzureSdkOwners: @mccoyp
```

Jim is in the process of updating the parser to be case-insensitive, but currently any case mismatch is treated like a comment and won't flag with the linter.   All of the tokens should use the mixed case in the suggestion to match.
_(I made this error in my PR and have to go back and adjust case on all of these elements)_